model-optimizer/mo/utils/versions_checker.py

```diff
@@ -160,11 +215,25 @@ def check_requirements(framework=None):
     """
     if framework is None:
         framework_suffix = ""
+    elif framework == "tf":
+        try:
+            exec("import tensorflow")
+            tf_version = sys.modules["tensorflow"].__version__
+            if tf_version < "2.0.0":
+                framework_suffix = "_tf"
+            else:
+                framework_suffix = "_tf2"
+            exec("del tensorflow")
+        except ImportError:
+            log.error("TensorFlow is not installed.")
+            return 1
```

Comment:
Why do we exit from function here? I would rather fallback to default behaviour `framework_suffix = "_tf"` and continue requirements checking

Reply:
Fixed.